### ISSUE TITLE
Chinese full-width punctuation such as "，" and "？" are not contained in the vocab

### ISSUE THREAD

**Issue by acadaiaca** [google/sentencepiece#789]
I used about 35m Chinese corpus to generate the Chinese vocab, here is my command：
`spm_train --input=test.zh --model_prefix=spm_zh --vocab_size=32000 --character_coverage=1`
but the spm_zh.vocab doesn't contain Chinese full-width punctuation such as "，" ,  "？", "；", "：". Instead, the spm_zh.vocab contains English half-width punctuation such as "," ,  "?", ";", ":". 
In fact, most of the lines in my training corpus use Chinese full-width punctuation but not English half-width punctuation. And Chinese full-width punctuation is different with English half-width punctuation. For example,
`ch = '，'  `
`en = ','  `
`print('ch punc', ch.encode('utf-8')) `
`print('en punc', en.encode('utf-8')) `
`>>ch punc b'\xef\xbc\x8c'`
`>>en punc b','`
I change other Chinese corpus and change vocab_size, however, the problem is still not solved.
Should I set other hyperparameters？

**Comment by taku910** [comment]
sentencepiece normalizes the input with Unicode NFKC by default, which includes the full/half width punctuations.
You can suppress the normalization with ``--normalization_rule_name=identity``.  We can also define application dependent
normalization rules.

https://github.com/google/sentencepiece/blob/master/doc/normalization.md

**Comment by geolvr** [comment]
> sentencepiece normalizes the input with Unicode NFKC by default, which includes the full/half width punctuations. You can suppress the normalization with `--normalization_rule_name=identity`. We can also define application dependent normalization rules.
> 
> https://github.com/google/sentencepiece/blob/master/doc/normalization.md

If I already have a trained model ‘xxx.model’, is it possible to modify its normalization settings instead of re-training? For example, can I pass some arguments into spm.EncodeAsIds() or spm.Load().
This is useful when loading Chinese language models pretrained on large generic datasets, like ChatGLM-6B.
